docOps/docs/libraries/nodejs/api_reference.md

```diff
@@ -387,131 +394,105 @@ Send the remainder value to its original address.
-| ------------------- | -------------------------------------- | ---------------------- | -------------------------------------------------------------------------------------------------------- |
-| [network]           | <code>number</code>                    | <code>undefined</code> | The tangle network to connect to (Mainnet = 1, Devnet = 1, Comnet = 3)                                   |
-| [primaryNode]       | <code>NodeUrl | [Node](#node)</code>   | <code>undefined</code> | A node URL to alway connect to first                                                                     |
-| [primaryPoWNode]    | <code>NodeUrl | [Node](#node)</code>   | <code>undefined</code> | A node URL to alway connect to first when using remote PoW, will be used before primaryNode              |
-| [node]              | <code>NodeUrl | [Node](#node)</code>   | <code>undefined</code> | A node URL to connect to                                                                                 |
-| [nodes]             | <code>NodeUrl | [Node](#node)[]</code> | <code>undefined</code> | A list node URL to connect to                                                                            |
-| [quorumSize]        | <code>number</code>                    | <code>undefined</code> | If multiple nodes are provided, quorum size determines the number of nodes to query to check for quorum. |
-| [quorumThreshold]   | <code>number</code>                    | <code>undefined</code> | Minimum number of nodes from the quorum pool that need to agree to consider a result true.               |
-| [localPow]          | <code>boolean</code>                   | <code>true</code>      | Whether to use local or remote PoW.                                                                      |
-| [MqttBrokerOptions] | <code>MqttBrokerOptions</code>         | <code>undefined</code> | Options for the MQTT broker                                                                              |
+| Field               | Type                             | Default                | Description                                                                                              |
+| ------------------- | -------------------------------- | ---------------------- | -------------------------------------------------------------------------------------------------------- |
+| [network]           | <code>number</code>              | <code>undefined</code> | The tangle network to connect to (Mainnet = 1, Devnet = 1, Comnet = 3)                                   |
+| [node]              | <code>string</code>              | <code>undefined</code> | A node URL to connect to                                                                                 |
+| [nodes]             | <code>string[]</code>            | <code>undefined</code> | A list node URL to connect to                                                                            |
```

Comment:
That's changed to be [Node](#node)[], could be a string or a node object with auth options